### ISSUE TITLE
Accessibility?

### ISSUE THREAD

**Issue by dotKev** [FancyGrid/FancyGrid#90]
Do you have any documentation on current accessibility features or a future roadmap regarding [WCAG](https://www.w3.org/WAI/standards-guidelines/)?

**Comment by MikhailTatsky** [comment]
Thank you for your question.
To be honest we have never studied this issue.
We will add it in TODO list.
We answered you on your email too.

**Comment by dotKev** [comment]
With the latest version of FancyGrid (1.7.73), we still get quite a few accessibility errors. We run our frontend output through [Pa11y](https://github.com/pa11y/pa11y) and consistently get errors.

```
 • Error: This textarea element does not have a name available to an accessibility API. Valid names are: label element, title undefined, aria-label undefined, aria-labelledby undefined.
   ├── WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.Textarea.Name
   ├── #fancy-grid-1 > textarea
   └── <textarea class="fancy-grid-copy-textarea"></textarea>

 • Error: This form field should be labelled in some way. Use the label element (either with a "for" attribute or wrapped around the form field), or "title", "aria-label" or "aria-labelledby" attributes as appropriate.
   ├── WCAG2AA.Principle1.Guideline1_3.1_3_1.F68
   ├── #fancy-grid-1 > textarea
   └── <textarea class="fancy-grid-copy-textarea"></textarea>

 • Error: This element's role is "presentation" but contains child elements with semantic meaning.
   ├── WCAG2AA.Principle1.Guideline1_3.1_3_1.F92,ARIA4
   ├── #fancy-grid-1 > div:nth-child(2) > div:nth-child(1)
   └── <div role="presentation" class="fancy-grid-header" style="height: 45px; width: 710px;"><div role="columnheader" class=...</div>

```

**Comment by MikhailTatsky** [comment]
@dotKev 
 Thank you for more details about this issue.
We need several days to study it.

**Comment by MikhailTatsky** [comment]
@dotKev 
 Thank you for providing us information about Pa11y.
Now we see a lot of issues.
We will be fixing it step by step.
